cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -80,6 +81,13 @@ type CMServer struct {
 	ServiceAccountKeyFile   string
 	RootCAFile              string
 
+	// volumeConfig
+	PersistentVolumeRecyclerDefaultScrubPod          string
+	PersistentVolumeRecyclerMinTimeoutNfs            int
```

Comment:
How do you suggest I pass CLI flags to volumes?  

We can code 1 config type of struct, bind CLI vars to it, and pass it down or we can add a map to VolumeHost with config where CLI flags are the keys.  Each plugin knows how to get what it wants/needs WRT configuration.